File: N-En-Raya-Profe/src/TipoFicha.java
import org.fusesource.jansi.Ansi;

import static org.fusesource.jansi.Ansi.Color.GREEN;
import static org.fusesource.jansi.Ansi.Color.RED;
import static org.fusesource.jansi.Ansi.ansi;

public enum TipoFicha {
    O,X;

    TipoFicha() {
    }

    public Ansi toAnsi() {
        Ansi OAnsi,XAnsi;

        OAnsi = ansi().fg(GREEN).a(O).reset();
        XAnsi = ansi().fg(RED).a(X).reset();

        return this.equals(O) ? OAnsi : XAnsi;
    }
}

File: N-En-Raya/src/TipoFicha.kt
import org.fusesource.jansi.Ansi

import org.fusesource.jansi.Ansi.Color.GREEN
import org.fusesource.jansi.Ansi.Color.RED
import org.fusesource.jansi.Ansi.ansi

enum class TipoFicha {
    O,X;

    fun toAnsi() : Ansi {
        val oAnsi = ansi().fg(GREEN).a(O).reset()
        val xAnsi = ansi().fg(RED).a(X).reset()

        return if (this == O) oAnsi else xAnsi
    }


}